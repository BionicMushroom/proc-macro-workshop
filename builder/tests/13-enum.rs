use derive_builder::Builder;

#[derive(Builder)]
enum Enum {
    Option1,
    Option2,
}

fn main() {}
