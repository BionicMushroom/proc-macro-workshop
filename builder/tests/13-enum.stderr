error: expected struct, found `enum`
 --> tests/13-enum.rs:4:1
  |
4 | enum Enum {
  | ^^^^
